src/shogun/distributions/HMM.h

```diff
@@ -67,6 +67,23 @@ typedef T_STATES* P_STATES ;
 
 //@}
 
+/**
+ * Developer's Note :
+ * - Changing the default Training type which is standard baum welch currently, would require to change 
+ *   the type passed to baum_welch_viterbi_train(BW_NORMAL) in train(CFeatures* data).
+ */
+
+/**
+ * @brief
+ * All currently supported Training types, with a default
+ * standard baum welch(BW_NORMAL), which will be used for training the given data.
+ *
+ * The enum defines various training algorithms types which need to be passed as argument (currently BW_NORMAL)
```

Comment:
please make doxygen reference the enum definitions, see
http://stackoverflow.com/questions/2335091/how-do-i-get-doxygen-to-link-to-enum-defintions